builds/securing-builds-by-strategy.adoc

```diff
@@ -0,0 +1,55 @@
+[id='securing-builds-by-strategy']
+= Securing builds by strategy
+include::modules/common-attributes.adoc[]
+:context: securing-builds-by-strategy
+toc::[]
+
+Builds in {product-title} are run in privileged containers. Depending on the
+build strategy used, this allows a user who can run builds to escalate their
+permissions on the cluster and host nodes. As a security measure, limit who can
+run builds and the strategy that is used for those builds. Custom builds are
+inherently less safe than Source builds, given that they can execute any code
+within a privileged container, and as such are disabled by default. Docker build
+permissions should also be granted with caution, as a vulnerability in the
+Dockerfile processing logic could result in a privileges being granted on the
+host node.
+
+By default, all users that can create builds are granted permission to use the
+Docker and Source-to-Image (S2I) build strategies. Users with *cluster-admin*
+privileges can enable the Custom build strategy, as referenced in the
+Restricting Build Strategies to a User Globally section.
+
+You can control who can build, and with which build strategy, using an
```

Comment:
s/build, and with which build strategy, using/build and which build strategies they can use by using
This seems too important for an aside. :man_shrugging:

Reply:
it's kinda the point of this whole section/topic (securing builds by strategy) so this doesn't strike me as an side so much a specific elaboration on what that means.

Reply:
It sounds like the two are of approximately equal importance, which is why I'd put them on equal footing in the sentence. You're right that the construction can be used for elaboration, but it's supposed to be elaboration that could be omitted without changing the meaning of the sentence.